src/UBC Toboggan/Assets/Scripts/boostLevelManager.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class boostLevelManager : MonoBehaviour
+{
+
+    public float minBoostPosition = -1.02f;
+    public Transform boostBar;
+
+    GameObject player;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        player = GameObject.FindWithTag("Player");
+    }
+
+    // Update is called once per frame
+    void Update()
+    { 
+        playerManager playerManagerScript = player.GetComponent<playerManager>();
+        float boostAmount = playerManagerScript.boostAmount;
+        float maxBoost = playerManagerScript.maxBoost;
```

Comment:
`playerManagerScript.maxBoost` is a constant, so you could make `maxBoost` a field in this class and move this line into `Start()`